src/software/logger/coloured_cout_sink.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <g3log/logmessage.hpp>
+#include <iostream>
+
+/**
+ * This class acts a custom sink for g3log
+ */
+class ColouredCoutSink
+{
+   public:
+    // Linux xterm foreground color
+    // http://stackoverflow.com/questions/2616906/how-do-i-output-coloured-text-to-a-linux-terminal
+    enum FG_Colour
+    {
+        YELLOW = 33,
+        RED    = 31,
+        GREEN  = 32,
+        WHITE  = 37
+    };
```

Comment:
hm, we could make it an enum class and use it as `FG_Colour::YELLOW` for more clarity?

Reply:
switched it to use MAKE_ENUM and a string conversion function